infra-in-containers/john.md

```diff
@@ -0,0 +1,272 @@
+## From VM to Container
+### Differences between a VM and a Container
+| VM | Container |
+| ---- | ---- |
+| Memory is allocated on setup | Memory is utilized on demand |
```

Comment:
Memory for containers can also be allocated during setup but the constraints on containers tend to be more elastic.